server/routes/homepage/index.ts

```diff
@@ -17,6 +18,12 @@ export default function routes(services: Services): Router {
       const { user } = res.locals
       const language = req.language || i18next.language
 
+      await auditService.sendAuditMessage({
+        action: 'VIEW_HOMEPAGE',
+        who: user.idToken.sub,
+        service: 'hmpps-launchpad-home-ui',
```

Comment:
Doesn't this want pulling from the helm values into config.ts and then into here as this is hardwired.